kola/README.md

```diff
@@ -0,0 +1,192 @@
+# Adding Tests
+
+
+## Quick Start
+
+1. Fork and clone this [`mantle` repository](https://github.com/coreos/mantle/)
+2. Move into `kola/tests/` and look for package which your test would fit best
+3. Edit the file and add your test(s), ensuring that you register your new test(s) in the packages `init()`
+4. Commit, push, and PR your result
+
+### Example
+Say we wanted to add a simple [noop](https://en.wikipedia.org/wiki/NOP_(code)) test in the `podman` test package. If we follow the above instructions it would look like this:
+
+```
+$ git clone git@github.com:$GITHUB_USERNAME/mantle.git
+<snip/>
+$ pushd kola/tests/
+$ $EDITOR podman/podman.go  # Add the test
+// Test: I'm a NOOP!
+func podmanNOOP(c cluster.TestCluster) {
+    // NOOP!
+}
+$ $EDITOR podman/podman.go # Register the test in the init
+func init() {
+    register.Register(&register.Test{
+        Run:         podmanNOOP,
+        ClusterSize: 1,
+        Name:        `podman.noop`,
+        Distros:     []string{"rhcos"},
+    })
+<snip/>
+$ popd
+$ ./build kola
+# Check and ensure the test is there
+$ ./bin/kola list | grep podman
+podman.base                                     [all]                                   [all]   [rhcos]
+podman.network                                  [all]                                   [all]   [rhcos]
+podman.noop                                     [all]                                   [all]   [rhcos]
+podman.workflow                                 [all]                                   [all]   [rhcos]
+# Run your test and see what happens
+$ sudo ./bin/kola run -b rhcos --qemu-image `pwd`/rhcos-410.8.20190502.0-qemu.qcow2 podman.noop
+=== RUN   podman.noop
+--- PASS: podman.noop (21.08s)
+PASS, output in _kola_temp/qemu-2019-05-08-1535-16606
+# git add/commit/push...
+# Open PR to get the test added!
+```
+
+## Grouping Tests
+
+Sometimes it makes sense to group tests together under a specific package, especially when these tests are related and require the same test parameters. For `kola` it only takes a forwarding function to do testing groups. This forwarding function should take `cluster.TestCluster` as it's only input, and execute running other tests with `cluster.TestCluster.Run()`.
```

Comment:
Worth noting that these steps are ran sequentially and on the same machine. As such, this is not recommended for any groups of tests that modify system state.

Additionally, the `FailFast` flag can be enabled during the test registration to skip any remaining steps after a failure has occurred.

Reply:
Add in this almost verbatim.